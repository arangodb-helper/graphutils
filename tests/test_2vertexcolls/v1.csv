_key|Hello|smart
1|1|A
2|2|B
