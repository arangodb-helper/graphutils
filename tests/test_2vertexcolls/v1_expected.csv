_key|Hello|smart
A:1|1|A
B:2|2|B
